src/ui/screen_process.c

```diff
@@ -34,6 +31,23 @@ void ui_screen_render_component(component_t* component)
     UG_SendBuffer();
 }
 
+/**
+ * Detects if the screen component being displayed has changed
+ * since the last time this function was called.
+ * This stores the last observed component into a global.
+ *
+ * @param[in] current_comp Current on-screen component.
+ */
+static bool _screen_has_changed(const component_t* current_comp)
```

Comment:
please no short names, as explicit long names are easier to understand, is less ambiguous etc.. `current_component`

Reply:
Fixed